templates/web/pages/tag/tag.tt.html

```diff
@@ -48,6 +50,42 @@
             [% description %]
         </div>
     [% END %]
+    [% IF tagid.defined  %]
+        <!-- injecting facet-knowledge-panel -->
+        <div id="facet-knowledge-panel">
+            <h2 id="title"></h2>
+            <P id="facet-panels"></P>
```

Comment:
It's strange to put it in a <p> tag, <div> is probably  more appropriate. In lowercase please. :)